README.md

```diff
@@ -145,6 +149,45 @@ Github Actions handles generating the `.rst` source files, generating the HTML s
 
 To view the documentation _locally_, go into the `/docs` directory and run `make html`. This will generate the .rst source files and build the static site in `/docs/build/html`
 
+## Code Style
+
+Code style on this project is linted using [pre-commit](https://pre-commit.com/). This python application is included in our `requirements-dev.txt` file, so if you have
+the applications requirements installed it should be available. pre-commit is run automatically on each push and PR by our [CI System](#continuous-integration).
+
+You can run it manually on all files with the command: `pre-commit run --all-files`.
+
+You can also set it up, so that it runs automatically for you on each commit. Running the command `pre-commit install` will install the pre-commit script in your
+local repositories git hooks folder, so that pre-commit is run automatically on each commit.
+
+### Configuration
+
+The pre-commit configuration file is named [`.pre-commit-config.yaml`](.pre-commit-config.yaml). This file configures the differnet lints that pre-commit runs.
+
+### Linters
+
+#### Built in
+
+Pre-commit ships with a [number of lints](https://pre-commit.com/hooks.html) out of the box, we are configured to use:
+- `trailing-whitespace` - trims trailing whitespace.
+- `end-of-file-fixer` - ensures that a file is either empty, or ends with one newline.
+- `check-yaml` - checks yaml files for parseable syntax.
+- `check-json` - checks json files for parseable syntax.
+- `check-ast` - simply checks whether the files parse as valid python.
+- `check-shebang-scripts-are-executable` - ensures that (non-binary) files with a shebang are executable.
+- `check-merge-conflict` - checks for files that contain merge conflict strings.
+- `check-added-large-files` - prevents giant files from being committed.
+- `mixed-line-ending` - replaces or checks mixed line ending.
```

Comment:
It doesn't look like `check-merge-conflict` is actually configured.

Reply:
Is now!